.idea/.gitignore

```diff
@@ -0,0 +1,10 @@
+# # Default ignored files
+# /shelf/
+# /workspace.xml
+# # Datasource local storage ignored files
+# /dataSources/
+# /dataSources.local.xml
+# # Editor-based HTTP Client requests
+# /httpRequests/
+.vscode/
+.idea/
```

Comment:
Is this file necessary?

Reply:
> Looking good! Mind also opening a pull request in Dalton that uses this branch and the pluck functionality?

I guess first we need to merge it to master? and create a tag. After that change Dalton's Gemfile probably.